### PR TITLE
Add more details about vulnerability refactor to changelog

### PR DESCRIPTION
**Pull request by avanbrunt-cb**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [ ] New and existing tests pass locally with the changes.
- [ ] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

N/A

## Pull Request Description

Add details about the refactor to vulnerabilities in the changelog for 1.3.0

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How Has This Been Tested?

N/A

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-  * Fix bug in helpers read_iocs functionality
-
 New Features
 
-* Added User Management, Grants, Access Profiles, Permitted Roles
-* Refactor of the vulnerability models
+* Add User Management, Grants, Access Profiles, Permitted Roles
+* Mov Vulnerability models to Platform package in preparation for supporting Endpoints and Workloads
+* Refactor Vulnerability models
+  * ``VulnerabilitySummary.get_org_vulnerability_summary`` static function changed to ``Vulnerability.OrgSummary`` model with query class
```
Comment (kebringer-cb) [inline docs/changelog.rst:11]: Formatting didn't work - think it needs a newline between the first and then second level bullet

File: docs/changelog.rst [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
+  * ``VulnerabilitySummary`` model moved inside ``Vulnerability`` to ``Vulnerability.AssetView`` sub model
+  * ``OrganizationalVulnerability`` and ``Vulnerability`` consolidated into a single model to include Carbon Black Cloud context and CVE information together
+  * ``Vulnerability(cb, CVE_ID)`` returns Carbon Black Cloud context and CVE information
+  * ``DeviceVulnerability.get_vulnerability_summary_per_device`` static function moved to ``get_vulnerability_summary`` function on ``Device`` model
+  * ``affected_assets(os_product_id)`` function changed to ``get_affected_assets()`` function and no longer requires ``os_product_id``
 * Add dashboard export examples
 * Live Response migrated from v3 to v6 (:doc:`migration guide<live-response-v6-migration>`)
```
Comment (kebringer-cb) [inline docs/changelog.rst:18]: Blank line after this for the sub-bullet to indent correctly

**Review by kebringer-cb — APPROVED** [review]
A few formatting changes

File: docs/changelog.rst [hunk 3]
```diff
@@ -1,22 +1,30 @@
...
-* Get Enriched Events for alert
+  * Live Response uses API Keys of type Custom
+* Add function to get Enriched Events for Alert
+
+Bug Fixes
+
+* Fix validate query from dropping sort_by for Query class
+* Fix the ability to set expiration for binary download URL
+* Fix bug in helpers read_iocs functionality
+
```
Comment (gyanitsa-cb) [inline docs/changelog.rst:27]: Should we add also the fix for device query?

**Review by emitreva-cb — APPROVED** [review]

**Review by gyanitsa-cb — APPROVED** [review]

**Pull request merged**